### PR TITLE
Configurable KubeApiServerIntegrator orphaned pods token bucket

### PR DESCRIPTION
**Pull request by tbak**

### CODE REVIEW

File: titus-common/src/main/java/com/netflix/titus/common/util/limiter/tokenbucket/internal/FixedIntervalTokenBucketSupplier.java [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+        return activeConfiguration.getCapacity() == configuration.getCapacity()
+                && activeConfiguration.getInitialNumberOfTokens() == configuration.getInitialNumberOfTokens()
+                && activeConfiguration.getIntervalMs() == configuration.getIntervalMs()
+                && activeConfiguration.getNumberOfTokensPerInterval() == configuration.getNumberOfTokensPerInterval();
+    }
+
+    private ActiveConfiguration reload() {
+        boolean same;
+        synchronized (lock) {
+            same = activeConfiguration != null && isSame();
```
Comment (joshi-keyur) [inline titus-common/src/main/java/com/netflix/titus/common/util/limiter/tokenbucket/internal/FixedIntervalTokenBucketSupplier.java:73]: Should the null check be also part of the isSame method?

File: titus-common/src/main/java/com/netflix/titus/common/util/limiter/tokenbucket/FixedIntervalTokenBucketConfiguration.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.titus.common.util.limiter.tokenbucket;
+
+import com.netflix.archaius.api.annotations.DefaultValue;
+
+public interface FixedIntervalTokenBucketConfiguration {
```
Comment (joshi-keyur) [inline titus-common/src/main/java/com/netflix/titus/common/util/limiter/tokenbucket/FixedIntervalTokenBucketConfiguration.java:21]: Should we add Configuration prefix for better namespacing here?

Comment (tbak) [reply titus-common/src/main/java/com/netflix/titus/common/util/limiter/tokenbucket/FixedIntervalTokenBucketConfiguration.java:21]: Prefix must be provided by a client, but we can create a default value here.

File: titus-common/src/main/java/com/netflix/titus/common/util/limiter/tokenbucket/internal/FixedIntervalTokenBucketSupplier.java (hunk 1 above)
Comment (tbak) [reply titus-common/src/main/java/com/netflix/titus/common/util/limiter/tokenbucket/internal/FixedIntervalTokenBucketSupplier.java:73]: Good idea. I will change it as suggested.

**Pull request merged**